src/sftp/handler.go

```diff
@@ -32,10 +32,10 @@ func NewSftpHandler() (*SftpHandler, error) {
 		return nil, err
 	}
 
-	pem, err := getPublicKeysForSshClient(credentialGetter)
-	if err != nil {
-		return nil, err
-	}
+	//pem, err := getPublicKeysForSshClient(credentialGetter)
```

Comment:
Can we leave a TODO or have a task in a card to uncomment this when necessary?

Reply:
Added todo!